### ISSUE TITLE
NTR: zinc:bicarbonate symporter activity and cadmium:bicarbonate symporter activity

### ISSUE THREAD

**Issue by lbreuza** [geneontology/go-ontology#18345]
Hi,
could you please consider creating the above mentioned terms

Definition :
Enables the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: Cd2+(out) + HCO3-(out) = Cd2+(in) + HCO3-(in). 
OR
Zn2+(out) + HCO3-(out) = Zn2+(in) + HCO3-(in).

as child of a new term that would be solute:bicarbonate symporter activity, itself a child of GO:0015294 solute:cation symporter activity and GO:0015106 bicarbonate transmembrane transporter activity.

Best, Lionel

**Comment by lbreuza** [comment]
Hi,
related to the same ticket and same protein, PubMed=27166256, showed that Slc39a8 is a transporter allowing the following symport :
Zn2+(out) + HCO3-(out) + selenite = Zn2+(in) + HCO3-(in) +selenite.
Could you please create a GO term for this molecular function?

Alternatively, we could create a term for selenite symporter activity, but that is not coupled to proton symport as described by  GO:0097079 selenite:proton symporter activity

I do not know what would be the best.
Cheers, Lionel

**Comment by pgaudet** [comment]
@lbreuza all all these functions described in PubMed=27166256? 

How specific is the transporter for the cation ? Perhaps we indeed need selenite:cation antiporter activity ? 

Thanks, Pascale

**Comment by lbreuza** [comment]
@pgaudet There are several papers including PMID:18037372 that describe the symporter activity of SLC39A8.
You can find them already in P2GO for that gene.

**Comment by lbreuza** [comment]
Can transport several cations including Mg2+, Fe2+, Hg2+ and Cd2+. So not very specific. However, the symport coupled to selenite and bicarbonate has only been shown for Zn2+.

**Comment by lbreuza** [comment]
Hi,
I am not sure about the status of this ticket. I have some pending annotations and would like to know what to do with them. Best, Lionel

**Comment by hdrabkin** [comment]
Holidays;

**Comment by pgaudet** [comment]
@lbreuza Are you sure this is a symporter ? I see in the paper that the Zn transport is dependent on zinc and bicarbonate but I don't see that they tested where these other ions end up ? 
Could it be an antiporter ? Or just be more efficient in the presence of those ions ?

**Comment by lbreuza** [comment]
Hi @pgaudet,
here is the function I wrote from different supporting evidence: " Functions as an energy-dependent symporter, transporting through the membranes an electroneutral complex composed of a divalent metal cation, a bicarbonate and a selenite anion or yet a metal cation and two bicarbonate anions (PubMed:27166256, PubMed:31699897)".
Cheers, Lionel

**Comment by pgaudet** [comment]
'cadmium has no known biological function in higher organisms, a cadmium-dependent carbonic anhydrase has been found in marine diatoms.' From https://en.wikipedia.org/wiki/Cadmium

Is Cd transport relevant biologically ? Or has it just been tested ?

**Comment by pgaudet** [comment]
@lbreuza for selenite, is selenite being transported in the same direction as the other ions ? 'Zn2+(out) + HCO3-(out) + selenite = Zn2+(in) + HCO3-(in) +selenite.'

**Comment by pgaudet** [comment]
Never mind - from the paper it seems to be a symporter.

**Comment by pgaudet** [comment]
+[Term]
+id: GO:0140410
+name: solute:bicarbonate symporter activity
+namespace: molecular_function
+def: "Enables the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: solute(out) + HCO3-(out) = solute(in) + HCO3-(in)." [PMID:27166256]
+comment: Created new term from request: https://github.com/geneontology/go-ontology/issues/18345.
+intersection_of: GO:0015293 ! symporter activity
+intersection_of: transports_or_maintains_localization_of CHEBI:17544 ! hydrogencarbonate
+intersection_of: transports_or_maintains_localization_of CHEBI:24431 ! chemical entity
+created_by: pg
+creation_date: 2020-01-07T16:58:49Z
+
+[Term]
+id: GO:0140412
+name: zinc:bicarbonate symporter activity
+namespace: molecular_function
+def: "Enables the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: zinc(out) + HCO3-(out) = zinc(in) + HCO3-(in)." [PMID:27166256]
+comment: Created new term from request: https://github.com/geneontology/go-ontology/issues/18345.
+intersection_of: GO:0015293 ! symporter activity
+intersection_of: transports_or_maintains_localization_of CHEBI:17544 ! hydrogencarbonate
+intersection_of: transports_or_maintains_localization_of CHEBI:63056 ! zinc cation
+created_by: pg
+creation_date: 2020-01-07T18:43:02Z
+
+[Term]
+id: GO:0140413
+name: zinc:bicarbonate:selenite symporter activity
+namespace: Enables the transfer of a solute or solutes from one side of a membrane to the other according to the reaction: zinc(out) + HCO3-(out) + HO3Se-(out) = zinc(in) + HCO3-(in) + HO3Se-(out). {xref="PMID:27166256"}
+namespace: molecular_function
+comment: Created new term from request: https://github.com/geneontology/go-ontology/issues/18345.
+intersection_of: GO:0015293 ! symporter activity
+intersection_of: transports_or_maintains_localization_of CHEBI:17544 ! hydrogencarbonate
+intersection_of: transports_or_maintains_localization_of CHEBI:18170 ! selenic acid
+intersection_of: transports_or_maintains_localization_of CHEBI:63056 ! zinc cation
+created_by: pg
+creation_date: 2020-01-07T18:58:12Z

**Comment by lbreuza** [comment]
Hi Pascale,
concerning Cadmium, Cd is a toxic and carcinogenic nonessential metal (1), which can enter the body through the intestine, skin, and lung and accumulates in the kidney (1–3). The level of Cd in the environment has risen with advances in industrialization, and the role of Cd in human disease is of increasing concern. The mechanisms of Cd toxicity are poorly understood, although it is known that Cd exerts its effects intracellularly, and there are polypeptides such as metallothionein (4) and reduced glutathione (5) that bind Cd and afford protection.
There is also a mouse model that shows resistance to Cd toxicity that has been used to identify the gene involved in.
They tested its transport because it is relevant in terms of physiology (environment) and not as any positive cations. This is why I thought we might create a term for this transport but I do not know about the policies of the Consortium with respect to this.
Thanks for creating the other terms.

**Comment by pgaudet** [comment]
I think Cd (like rubidium and others) is 'hijacking' a transporter that transports similar substances, but the cell has not bona fide Cd transporter. 

This is considered outside the scope of GO. 

Thanks, Pascale

**Comment by deustp01** [comment]
"Hijacking" here seems to me like too narrow a view of an organism's short term responses to environmental stresses and of a population's long-term evolutionary responses to changing environments. Agreed, life downwind of a smelter may not fit our view of a normal stress or environment but it's also not the case that environmental cadmium or rubidium levels were the integer zero in our state of nature, and the otherwise edible plants that take up minerals like cadmium and concentrate them to human-toxic levels are not new.

**Comment by pgaudet** [comment]
Right - response to Cadmium would be a 'normal' function, and certainly exporting it would also make sense - and it would make sense to annotate the protein to something like 'xenobiotic detoxification by transmembrane export across the plasma membrane' (obviously Cd is not a xenobiotic, but transporting it out would detoxify it). 

But as far as I can tell from the paper this gene product does not export Cd, it 'accidentally' imports it. Is this right ?

**Comment by lbreuza** [comment]
Yes @pgaudet. As far as I understand cadmium is accidentally symported and toxic for cells. Some mice strain that express less transporter are less susceptible to Cd toxicity. Not easy to draw a line between what is physiological and what is not here.
Lionel

**Comment by pgaudet** [comment]
GO:0140410  solute:bicarbonate symporter activity 
should be labelled monoatomic cation:bicarbonate symporter activity 

this is more accurate for the children it's trying to group
